5_vizprep/out/storm_point_[20170826-11]_fun.rds.ind

```diff
@@ -0,0 +1,2 @@
+indication_time: 2018-07-11 13:13:12 UTC
```

Comment:
but since david just resolved https://github.com/USGS-R/scipiper/issues/54, i'll leave this in here so we can delete the other ones and rebuild this on in the future. expecting a change from `indication_time` to `hash` in that future version of this file; i assume indication_time was from some earlier version of the function that created this file.

Reply:
I'd like to go back to the `[time]` notation because I think it makes it more explicit that we are parsing the content w/in those brackets and easier to read. I used `-` as a placeholder with the hope that I could change it back when that vizlab update was made.

Reply:
Yes. go for it.